### PR TITLE
Select collection exercise based on survey

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Show collection exercise based on the chosen survey.
Only displays surveys with at least one collection exercise.

### How to review 
Ensure:
- app works as expected.
- surveys are only shown with collection exercises.
- selecting a collection exercise displays details for chosen exercise.
- error is shown when reporting service is not reachable.

### CODE REVIEW

File: app/common/jinja_filters.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from flask import Blueprint
+
```
Comment (ryangrundy7) [inline app/common/jinja_filters.py]: Don't need it

**Review by ryangrundy7 — DISMISSED** [review]

File: app/static/assets/js/dashboard.js [hunk 2]
```diff
@@ -1,72 +1,138 @@
...
+        autoHeight: false,
+        scrollY: "30vh",
+        scrollCollapse: true,
+        scroller: {
+            loadingIndicator: false
+        },
+        data: [],
+        columns: [{
+            "data": "userDescription",
+            "title": "Collecton Exercise Period",
```
Comment (AdamHawtin) [inline app/static/assets/js/dashboard.js]: Spelling mistake

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by ryangrundy7 — CHANGES_REQUESTED** [review]
Refreshing a page turns into an empty page

**Review by AdamHawtin — APPROVED** [review]

**Review by ryangrundy7 — APPROVED** [review]

**Comment by AdamHawtin** [comment]
[Selecting the exercise once in a survey](https://trello.com/c/ItzBKaCm/33-selecting-the-exercise-once-in-a-survey)

**Pull request merged**